Minisat/core/Dimacs.h

```diff
@@ -43,7 +45,7 @@ static void readClause(B& in, Solver& S, vec<Lit>& lits) {
     for (;;){
         parsed_lit = parseInt(in);
         if (parsed_lit == 0) break;
-        var = abs(parsed_lit)-1;
+        var = std::abs(parsed_lit)-1;
```

Comment:
Another inconsistency with this one. Although, since Minisat has mostly been just "copy-pasted" from it's original source, simply not touching it, as long as it works, could also be a strategy. I also doubt we include `Dimacs.h` anyway, but that's not a point...

Reply:
I had one test(tQuotientE.cpp) failing compiled with GCC if `abs` is without namespace qualification.
So I added `std::` to all abs to be safe.